example/cli/odp_cli.c

```diff
@@ -161,9 +172,30 @@ int main(int argc, char *argv[])
 		exit(EXIT_FAILURE);
 	}
 
-	/* Start CLI server. */
-	if (odph_cli_start()) {
-		ODPH_ERR("CLI start failed.\n");
+	/* Create server thread. */
+
+	odp_cpumask_t cpumask;
+	odph_thread_common_param_t thr_common;
+	odph_thread_param_t thr_param;
+	odph_thread_t thr_server;
+
+	if (odp_cpumask_default_control(&cpumask, 1) != 1) {
+		ODPH_ERR("Failed to get default CPU mask.\n");
+		exit(EXIT_FAILURE);
+	}
+
+	memset(&thr_common, 0, sizeof(thr_common));
```

Comment:
PR 1355 adds new param init functions. It would be good to merge this PR after that one, and use those new functions here.